src/setup/platformSetup/index.website.js

```diff
@@ -6,7 +6,7 @@ import 'shim-keyboard-event-key';
 import checkForUpdates from '../../libs/checkForUpdates';
 import Config from '../../CONFIG';
 import DateUtils from '../../libs/DateUtils';
-import {version as currentVersion} from '../../../package.json';
```

Comment:
Is this change necessary?

Reply:
Webpack will only allow ESM to have named imports. So non-ESM can only be doing 'default' imports 

[Here is the reference from Webpack 5 docs](https://webpack.js.org/guides/ecma-script-modules/#flagging-modules-as-esm:~:text=Only%20the%20%22default%22%20export%20can%20be%20imported%20from%20non%2DESM.%20Named%20exports%20are%20not%20available)

<img width="1309" alt="Screenshot 2022-08-08 at 22 11 44" src="https://user-images.githubusercontent.com/33981371/183518048-6d0bdbda-829f-49a3-89e8-58303f20535e.png">